guides/common/modules/ref_system-requirements.adoc

```diff
@@ -30,6 +27,16 @@ In addition, a minimum of 4 GB RAM of swap space is also recommended.
 endif::[]
 endif::[]
 
+ifdef::foreman-deb[]
+* The latest version of a supported operating system
+endif::[]
+ifndef::foreman-deb,satellite[]
+* The latest version of {EL} 8 or {EL} 7 Server
+endif::[]
+ifdef::satellite[]
+* The latest version of {EL} 8
+endif::[]
```

Comment:
@ekohl I'm wondering if it would be acceptable to say "The latest version of a supported operating system" for all builds, when we have a special section about supported OSs. It would minimize the occurence of particular version numbers --> easier maintenance.

Reply:
I like avoid the specific number to make it generic. How about `The operating system should have all available updates applied`?